phac-nml/irida-next
# Local time

## What does this PR do and why?
This PR adjusts `datetimes` on the website to use the user's local time. This includes removing our previous `viral_time_ago` component and using `local_time` gem's `time_ago`

Datepicker will be updated in a future PR

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/6377ff4c-bdb8-433c-831f-878770df7fb4)

![image](https://github.com/user-attachments/assets/5d3dd5df-d9b7-45e2-8249-a42b07b9200f)


## How to set up and validate locally
1. Navigate throughout the website wherever there is a timestamp, and verify the time is adjusted to your timezone.
This includes:
- Project/Group listing table
- Members table
- Samples table
- History
- Workflow Executions table
- Data Exports table
- Bots table
- PAT table

2. Navigate to Data Exports Mailer preview (http://localhost:3000/rails/mailers/data_export_mailer/). Ensure the datetime shown for expiry includes `23:59 UTC`

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
